src/scripts/classes/Cell.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+class Cell {
+  constructor(element, value) {
+    this._element = element;
+    this._value = value || 0;
+  }
+
+  get value() {
+    return this._value;
+  }
+
+  set value(number) {
+    this._value = number;
+    this._element.innerText = number || ''; // ignore zero
+
+    this._element.className = this._element.className
+      .replace(/( ?field__cell--\d+)/, '');
+    this._element.classList.add(`field__cell--${number}`);
+    this.state = '';
+  }
+
+  set state(state = '') {
+    void this._element.offsetWidth; // trick to restart animation
```

Comment:
if your code need comments

it is bad code

Reply:
@danylo-onishchuk I agree with you, but not totally. Self-describle fucntions and variables are best practice, but in some cases I prefer to use comments to organize code structure or put some small notes. As for me good code is a balance between naming and commenting.